test-vectors/message.json

```diff
@@ -0,0 +1,32 @@
+{
+  "class": "message",
+  "selector": "<some predicate for the driver to determine if this vector applies to the implementation, e.g. to deal with protocol upgrades>",
+
+  "_meta": {
+    "id": "test_vector_id",
+    "version": "version_id",
+    "gen": {
+      "source": "lotus",
+      "version": "0.4.1+git.27d74337+api0.8.1"
+    }
+  },
+
+  "preconditions": {
+    "state_tree": {
+      "t01": "0x0123456789abcdef",
+      "t02": "0x0123456789abcdef",
+      "t03": "0x0123456789abcdef"
+    }
```

Comment:
@willscott this is where we'll set the plucked/masked state tree.

@nonsense this is the state tree we need to slip in the genesis block.

Reply:
This could be a serialized CAR, but I thought we might as well set it in map form, because:
1. it's more obvious, human-readable, and easier to debug.
2. we control this input entirely (as it's generated by the generators).
3. VM implementation may have custom mechanics for state imports, and this is the most explicit choice, which makes it a sane fallback/common denominator.

Reply:
I am not sure what is the best format to specify the input state tree (just because I don't have a good understanding of what exactly is the state right now) - serialised CAR would definitely work, I am not sure about individual key/values and whether it makes sense to do that. Also here you specify t01 t02 t03 - these are not CIDs, I assume you mean that this is the state of the miners, or?